integration-tests/data/contract_origination.ts

```diff
@@ -1,69 +1,5 @@
 import { OperationContentsOrigination } from "@taquito/rpc"
 
-export const contractOriginationParams = {
```

Comment:
was this related to toru? wondering why it was removed

Reply:
line12 tx_rollup_l2_address don't think it's possible to originate this contract since it's disabled